frontend/appflowy_flutter/lib/user/presentation/router.dart

```diff
@@ -138,26 +151,43 @@ class SplashRoute {
     UserProfilePB userProfile,
     WorkspaceSettingPB workspaceSetting,
   ) {
-    Navigator.push(
-      context,
-      PageRoutes.fade(
-        () => HomeScreen(
-          userProfile,
-          workspaceSetting,
-          key: ValueKey(userProfile.id),
+    if (Platform.isAndroid || Platform.isIOS) {
```

Comment:
Would it be better if we replace all existing Platform.isAndroid || Platform.isIOS to a single function?